### ISSUE TITLE
Cannot read property 'resource' of null in production. 

### ISSUE THREAD

**Issue by andrewtch** [fqborges/webpack-fix-style-only-entries#2]
when running wp4 with --mode production, the error above occurs. 

Seems  to be magically fixed by changing 

```
resources = modules.map(m => m.resource);
```

to 

```
              modules.every(m => {
                if (m && m.resource) {
                  resources.push(m.resource);
                }
              })
```

**Comment by fqborges** [comment]
@andrewtch I could not reproduce, but made a fix for the case any mode is null or its resource is null. Already published on npm at version 0.0.3.

Webpack documentation is very light on details. Would you mind share your webpack config for this production build so I can understand why it happens?

Thanks for reporting.

**Comment by fqborges** [comment]
I closed by inactivity, since it is fixed on 0.0.3
